x-pack/plugins/fleet/server/routes/setup/index.ts

```diff
@@ -25,7 +29,7 @@ export const registerFleetSetupRoute = (router: FleetAuthzRouter) => {
     .addVersion(
```

Comment:
`fleetSetupHandler` is registered under both `/setup` and `/agents/setup` paths, the latter was not documented in openapi, is it used anywhere?